Framework/DataHandling/src/LoadGSS.cpp

```diff
@@ -253,9 +262,9 @@ API::MatrixWorkspace_sptr LoadGSS::loadGSASFile(const std::string &filename,
         std::vector<double> storeY = vecY;
         std::vector<double> storeE = vecE;
 
-        gsasDataX.push_back(storeX);
-        gsasDataY.push_back(storeY);
-        gsasDataE.push_back(storeE);
+        gsasDataX.emplace_back(storeX);
+        gsasDataY.emplace_back(storeY);
+        gsasDataE.emplace_back(storeE);
```

Comment:
If you use move here, it might pay of the complement it with a `reserve` call for `vecX`, `vecY`, and `vecE`, to speed up the `push_backs` done below.